### PR TITLE
fixed install script

### PR DESCRIPTION
**Pull request by damienchallet**
A fix to fix both issues I reported

### CODE REVIEW

**Comment by Evidlo** [comment]
Thanks.  Why is the `cd /home/root` necessary?  What if the user copies the install script and mount file to `/tmp/` instead of `/home/root`?

**Comment by damienchallet** [comment]
Indeed, but the point is that your install instructions imply that the
install files end up in /home/root

File: entware_install.sh [hunk 1]
```diff
@@ -84,6 +85,7 @@ fi
 /opt/bin/opkg upgrade
 
 # create systemd mount unit to mount over /opt on reboot
+cd /home/root
 cp opt.mount /etc/systemd/system/
```
Comment (Evidlo) [inline entware_install.sh]: How about this instead:

``` bash
parent_dir=$(dirname $0)
cp $parent_dir/opt.mount /etc/systemd/system
```

**Review by Evidlo — CHANGES_REQUESTED** [review]

**Comment by damienchallet** [comment]
Sounds good !

**Review by Evidlo — APPROVED** [review]

**Comment by Evidlo** [comment]
Thanks

**Pull request merged**